doc/onlinehelp.rst

```diff
@@ -1,30 +1,93 @@
-Or, if you're not sure of the name of the object you want help on, or are looking for a specific keyword, use the :guilabel:`Search` field to get a list of results.
-You can view and re-run previous searches with the drop-down menu or the :kbd:`Down Arrow` key in the :guilabel:`Module or package field`, navigate previous pages with the forward and back buttons (top left, left and right arrows), and go back to the homepage with the home button (house icon top left) or the ``Module Index`` link top right.
+
+.. image:: images/online_help/online-help-name.gif
+   :alt: Spyder Online Help pane showing module browsing by name
+
+|
+
+Next to the title of the package, you will see the path of the location where the package was installed in your computer. 
+Click the path to see the page source code.
+
+.. image:: images/online_help/online-help-path.gif
+   :alt: Spyder Online Help pane showing source code clicking module's path
+
+|
+
+If you're not sure of the name of the object you want help on, or are looking for a specific keyword, use the :guilabel:`Search` field to get a list of results.
+
+.. image:: images/online_help/online-help-search.gif
+   :alt: Spyder Online Help pane showing search of a keyword
+
+|
+
+The Online Help also provides an index of topics that might be useful for your search and an index of keywords corresponding to reserved words in Python.
+
+
+
+===============
+Toolbar Options
+===============
+
+You navigate previous pages with the forward and back buttons (top left, left and right arrows), and go back to the homepage with the home button (house icon top left) or the ``Module Index`` link top right.
+
+.. image:: images/online_help/online-help-navigation.gif
+   :alt: Spyder Online Help pane showing navigation with arrows
+
+|
 
 Perform a realtime search within a page's content with the :guilabel:`Find` button (magnifying glass icon top left) or :kbd:`Ctrl-F`, navigate through matches with the Up and Down buttons, and make matching case sensitive with the :guilabel:`Aa` button.
+
+.. image:: images/online_help/online-help-find.gif
+   :alt: Spyder Online Help pane showing finding a word on a module
+
+|
+
+You can view and re-run previous searches with the drop-down menu or the :kbd:`Down Arrow` key in the :guilabel:`Module or package field`.
+
+.. image:: images/online_help/online-help-history.png
+   :alt: Spyder Online Help pane showing previous searches
+
+|
+
 You can use the zoom in/out buttons (:guilabel:`-` and :guilabel:`+`, top right), to change the font size to suit your preferences.
+
+.. image:: images/online_help/online-help-zoom.gif
+   :alt: Spyder Online Help pane showing changing the font
+
+|
+
 Finally, to cancel searching or page loading, click the stop button (red :guilabel:`X`, top right) and to reload the help content you're seeing to the most current version on disk (such as when you change your package's documentation), hit the refresh button (circular arrows, top right).
```

Comment:
```suggestion
To cancel searching or page loading, click the stop button (red :guilabel:`X`, top right), and to reload the page (such as when you change your own package's documentation), press the refresh button (circular arrows, top right).
```

My sentence here is way too long and complicated, as you've pointed out for me before, so I've simplified it.